src/components/FileDrop.vue

```diff
@@ -6,7 +6,7 @@
 				type="file"
 				name="file"
 				accept=".woff,.woff2,.ttf,.otf"
-				@change="getFont"
+				@change="$emit('getFont', $event)"
```

Comment:
Once we move on to `<script setup>` syntax we could use methods again (if there's value in it).
Requires to use the return value of the `emits` below (and not using the Options style of writing SFC).

One step at a time.

Reply:
Yeah, if it brings benefit. For now I'm happy that everything is fresh, clean and working so I'm fine with this.